warehouse/models/mart/gtfs_quality/fct_daily_rt_feed_validation_notices.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(materialized='table') }}
+
+WITH fct_daily_rt_feed_files AS (
+    SELECT * FROM {{ ref('fct_daily_rt_feed_files') }}
+),
+
+codes AS (
+    SELECT * FROM {{ ref('stg_gtfs_quality__rt_validation_code_descriptions') }}
+),
+
+outcomes AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_validation_outcomes') }}
+),
+
+notices AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_validation_notices') }}
+),
+
+fct_daily_rt_feed_validation_notices AS (
+    SELECT
+        {{ dbt_utils.surrogate_key(['daily_feeds.date', 'daily_feeds.base64_url', 'codes.code', 'codes.is_critical']) }} AS key,
+        daily_feeds.date,
+        daily_feeds.base64_url,
+        codes.code,
+        codes.is_critical,
+        -- TODO: at some point, these codes will be versioned by validator version
+        ARRAY_AGG(DISTINCT notices.gtfs_validator_version IGNORE NULLS) AS gtfs_validator_versions,
+        SUM(daily_feeds.parse_success_file_count) / COUNT(daily_feeds.key) AS parsed_files,
```

Comment:
I feel like this whole CTE could maybe use a few comments or something, this was incredibly hard for me to parse especially because it's aggregating across a bunch of different grains at once